securitas/templates/group.html

```diff
@@ -86,7 +86,7 @@ <h4>Sponsors</h4>
                 <div class="text-muted">
                   {% if current_user_is_sponsor %}
                     <button type="submit" class="btn btn-outline-danger border-0" name="username" value="{{ member.username }}">
-                      <i class="fa fa-trash"></i>
+                      <em class="fa fa-trash"></em>
```

Comment:
`<em>` is not the right replacement here. If you hate `<i>` for some reason, then `<span>` is the correct replacement here.